modules/nodes-scheduler-taints-tolerations-about.adoc

```diff
@@ -10,7 +9,7 @@
 
 A _taint_ allows a node to refuse pod to be scheduled unless that pod has a matching _toleration_.
 
-You apply taints to a node through the node specification (`NodeSpec`) and apply tolerations to a pod through the `Pod` specification (`PodSpec`). A taint on a node instructs the node to repel all pods that do not tolerate the taint.
+You apply taints to a node through the `Node` specification (`NodeSpec`) and apply tolerations to a pod through the `Pod` specification (`PodSpec`). A taint on a node instructs the node to repel all pods that do not tolerate the taint.
```

Comment:
The terminology for this feature is... interesting.

While _repel_ makes sense, referring to scheduling might be easier to read. Maybe

> When a taint is applied to a node, it can schedule only pods that are configured to tolerate the taint.

Even that probably isn't ideal.

Reply:
@jboxman 
I think I got this originally from the Kubernetes doc (I recall having a hard time understanding these concepts when I first wrote up these docs for 3.?. I was still a bit of a newbie.)
_When you apply a taint a node, the scheduler cannot place a pod on that node unless the pod can tolerate the taint._??